quick-start.md

```diff
@@ -10,6 +12,12 @@
 1. Save.
 1. Done.
 
+### Using Docker
+1. Clone the repo
+1. To start, run `docker-compose up --build`
+1. To stop, use `ctrl+c / cmd+c`
```

Comment:
I haven't double checked but I think you'll find that it's Ctrl+C on macOS, not ⌘+C

Reply:
Good spot, thanks!